### ISSUE TITLE
MassSave Incentive Autofiller

### ISSUE THREAD

**Issue by mzagaja** [codeforboston/cambridge_energy_app#12]
MassSave has a bunch of incentives and we want to see if we can make it easy to produce the filled out forms and get them sent. Is there a service that will print and mail these without human intervention or can we use an API fax service or e-mail endpoint?

**Comment by mzagaja** [comment]
@challyed would you be able to do this research and report back by Saturday with:

1. What incentives are available
2. The process for applying for each incentive that includes any intake capabilities via fax, e-mail, web form or USPS. Make sure to include relevant URLs.

Once this initial phase is done we can hand it off to a developer to build the appropriate wrapper/adapter.

**Comment by mzagaja** [comment]
Since nobody has self-assigned this I have kicked this to 0.2. If someone wants to take ownership of this and run with it for the next month I'm happy to pull it back into 0.1.

**Comment by mzagaja** [comment]
I think we're going to declare this feature out of scope.
